src/main/java/cubicchunks/worldgen/generator/custom/CustomGeneratorSettings.java

```diff
@@ -149,10 +149,13 @@
     public float diamondOreSpawnMaxHeight = -0.75f;
 
     public int lapisLazuliSpawnSize = 7;
-    public int lapisLazuliSpawnTries = 1;
-    public float lapisLazuliSpawnProbability = 256f / 32f / (256f / Cube.SIZE);
-    public float lapisLazuliHeightMean = 0.25f; // actually vanilla closest fit is 15/64 and 7/64
-    public float lapisLazuliHeightStdDeviation = 0.125f;
+    public int lapisLazuliSpawnTries = 1; //base value to 3 to avoid extremely rare spawns
+    public float lapisLazuliSpawnProbability = 0.933307775f; //256f / 16f / (256f / Cube.SIZE);
+    public float lapisLazuliHeightMean = -1.0f; // actually vanilla closest fit is 15/64 and 7/64
+    public float lapisLazuliHeightStdDeviation = 7.1882908513f;
```

Comment:
The old comments are very misleading here.
And all values should be scaled for world generation height-1 to 1 by default (-1 is vanilla height 0, 1 is vanilla height 128).

These values should be then multiplied and offset by the right values when it's used.

Reply:
The height scaling is in there already, you mentioned that on the last PR already.
I can change the comments though, that's right. I can also resubmit that for 1.12 too, the affected files didn't change I think.

Reply:
It's not there for `lapisLazuliHeightStdDeviation`.